src/sql/workbench/parts/objectExplorer/common/treeNode.ts

```diff
@@ -91,6 +91,8 @@ export class TreeNode {
 
 	public iconType: string | SqlThemeIcon;
 
+	public iconPath: string | { light: string, dark: string };
```

Comment:
Looking at the codebase it looks like VSCOde is moving to using Uri internally, and doing a conversion earlier from string | { light: string; dark: string; } to just standardizing on "process the contribution once, assign URIs for light and dark icons, then share that to simplify usage. Could we do this?

Reply:
fixed.